packages/indexer/src/handlers/events/substrateChains/handleGetRequestHandledEvent.handler.ts

```diff
@@ -0,0 +1,76 @@
+import { SubstrateEvent } from "@subql/types"
+import { Status } from "@/configs/src/types"
+import { getHostStateMachine } from "@/utils/substrate.helpers"
+import { HyperBridgeService } from "@/services/hyperbridge.service"
+import { Request } from "@/configs/src/types/models"
+import { GetRequestService } from "@/services/getRequest.service"
+
+type EventData = {
+	commitment: string
+	relayer: string
+}
+export async function handleSubstrateGetRequestHandledEvent(event: SubstrateEvent): Promise<void> {
+	logger.info(`Saw Ismp.GetRequestHandled Event on ${getHostStateMachine(chainId)}`)
+
+	if (!event.extrinsic && event.event.data) return
+
+	const {
+		extrinsic,
+		block: {
+			timestamp,
+			block: {
+				header: { number: blockNumber, hash: blockHash },
+			},
+		},
+	} = event
+
+	const eventData = event.event.data[0] as unknown as EventData
+	const relayer_id = eventData.relayer.toString()
+
+	logger.info(
+		`Handling ISMP GetRequestHandled Event: ${JSON.stringify({
+			data: event.event.data,
+		})}`,
+	)
+
+	const host = getHostStateMachine(chainId)
+
+	const request = await Request.get(eventData.commitment.toString())
+
+	if (!request) {
+		logger.error(`Get Request not found for commitment ${eventData.commitment.toString()}`)
+		return
+	}
+
+	let status: Status
+	if (request.dest === host) {
+		status = Status.DESTINATION
+	} else {
+		status = Status.HYPERBRIDGE_DELIVERED
+	}
```

Comment:
For GetRequestHandled. If the source != host then we should use `HYPERBRIDGE_DELIVERED`. These are requests processed by hyperbridge